docs/userGuide/components/tipBox.md

```diff
@@ -0,0 +1,100 @@
+# TipBox
+
+#### Alias: box

```

Comment:
The `Alias` seems out of place and as only 2 words. Maybe a short description saying you can use either `<tip-box>` or `<box>`?